scripts/run_e2e_tests.py

```diff
@@ -271,7 +256,6 @@ def build_js_files(dev_mode_setting, deparallelize_terser=False):
         deparallelize_terser: bool. Represents whether to use webpack
         compilation config that disables parallelism on terser plugin.
     """
-    update_dev_mode_in_constants_js(CONSTANT_FILE_PATH, dev_mode_setting)
```

Comment:
Shouldn't anything else be changed in this file? (e.g. if we want to run the e2e tests in dev vs prod)?

Or are we forcing one mode from here on? /cc @U8NWXD